README.md

```diff
@@ -266,14 +283,17 @@ az group delete --name demo-fastapi
 
 When deploying, you might encounter errors or problems, either on the automation part of it (GitHub Actions) or on the deployment destination (Azure Web Apps).
 
-If running into trouble, check logs in the portal or use the following with the Azure CLI:
+You can check the logs of the Github Actions workflow by selecting the latest workflow from the _Actions_ tab. Find the first step that has a broken icon next to it, and expand that step to see what went wrong in it.
```

Comment:
I had to debug in the Actions logs for all my issues, I think that should be first stop since it should also show Azure issues.